### PR TITLE
#134 Finalize TaxCategorySyncer

### PR DESCRIPTION
**Pull request by leungkinghin**
Issue : #134 

- Finalise all missing points in coverage
- Add Documentation
- Last Review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +33,7 @@ As of now, these are the supported resources:
 - ProductTypes
 - Types
 - States
+- TaxxCategories
```
Comment (ahmetoz) [inline README.md:36]:
```suggestion
- TaxCategories
```

File: README.md [hunk 2]
```diff
@@ -159,10 +164,11 @@ commercetools/commercetools-project-sync:3.3.0 -s all
...
  1. `Type` Sync and `ProductType` Sync and `States` Sync in parallel.
- 2. `Category` Sync.
- 3. `Product` Sync.
- 4. `CartDiscount` Sync.
- 5. `InventoryEntry` Sync.
+ 2. `TaxCategory` Sync.
+ 3. `Category` Sync.
+ 4. `Product` Sync.
+ 5. `CartDiscount` Sync.
+ 6. `InventoryEntry` Sync.
```
Comment (JudeNiroshan) [inline README.md:172]: Also include the docker command below. Just like the other sync types has already added,

File: README.md [hunk 3]
```diff
@@ -75,7 +79,8 @@ As of now, these are the supported resources:
    usage: commercetools-project-sync
     -h,--help               Print help information.
     -s,--sync <arg>         Choose one of the following modules to run: "types", "productTypes", "states", "categories", 
-                            "cartDiscounts", "products", "inventoryEntries" or "all" (will run all the modules).
```
Comment (ahmetoz) [inline README.md:78]: I would follow the order and put tax categories after the states.
`Choose one of the following modules to run: "types", "productTypes", "states", "taxCategories", "categories"... `

File: README.md (hunk 2 above)
Comment (leungkinghin) [reply README.md:172]: https://github.com/commercetools/commercetools-project-sync/pull/139/commits/92ba3bb67a2777e0a7dca1b0d4260f80723284ea

File: README.md (hunk 3 above)
Comment (leungkinghin) [reply README.md:78]: https://github.com/commercetools/commercetools-project-sync/pull/139/commits/f56b41e6d35e6d2753f27a8f25d948e335c2e4de

File: README.md (hunk 1 above)
Comment (leungkinghin) [reply README.md]: https://github.com/commercetools/commercetools-project-sync/pull/139/commits/f56b41e6d35e6d2753f27a8f25d948e335c2e4de

**Pull request merged**